modules/ROOT/pages/design-documentation.adoc

```diff
@@ -0,0 +1,305 @@
+
+The following example API specification demonstrates different ways, indicated by ten callouts (1 - 10), that you can set server definitions. After the example API specification, the documentation for the portion of the API at each callout is illustrated with a screenshot.
+
+----
+openapi: '3.0.2'
+info:
+  title: Multi-server Cemo API
+  version: '1.0'
+servers: <1>
+  - url: https://{customerId}.saas-app.com:{port}/v2
+    variables:
+      customerId:
+        default: demo
+        description: Customer ID assigned by the service provider
+      port:
+        enum:
+          - '443'
+          - '8443'
+        default: '443'
+  - url: https://{region}.api.company.com
+    variables:
+      region:
+        default: westus
+        enum:
+          - westus
+          - eastus2
+          - westcentralus
+          - westeurope
+          - southeastasia
+  - url: https://api.company.org/data/2.5/
+    description: Production server
+  - url: http://beta.api.company.org/data/2.5/
+    description: Beta server
+paths:
+  /default:<2>
+    description: A default path to the API
+    get:<3>
+      summary: "The get method for /default"
+      responses:
+        200:
+          description: Successful response
+  /duplicated:<4>
+    description: Has some of the default servers
+    servers:
+      - url: https://api.company.org/data/2.5/
+        description: Production server
+      - url: http://beta.api.company.org/data/2.5/
+        description: Beta server
+    get:<5>
+      summary: "The get method for /duplicated"
+      responses:
+        200:
+          description: Successful response
+  /files:<6>
+    description: File upload and download operations
+    servers:
+      - url: https://files.example.com
+        description: Override base path for all operations with the /files path
+    get:<7>
+      summary: "The get method for /files"
+      responses:
+        200:
+          description: Successful response
+  /ping:<8>
+    servers:
+      - url: https://endpoint.example.com
+        description: Override base path servers
+    get:<9>
+      servers:
+        - url: https://echo.example.com
+          description: Override base path for the GET /ping operation
+        - url: https://echo2.example.com
+          description: Also overrides base path for the GET /ping operation
+      summary: "The get method for /ping"
+      responses:
+        200:
+          description: Successful response
+    post:<10>
+      summary: "The post method for /ping"
+      responses:
+        201:
+          description: Successful create
+----
+
+1. The Summary page of the documentation for this API lists the four servers that are defined in this `servers` field:
++
+image::apid-apic-oas-3-summary.png[]
+
+2. The `/default` endpoint does not override the base `servers` definition at the top of the API. On the Overview page for the endpoint, you can select from those four servers to see the URI for this endpoint at each:
```

Comment:
I cannot parse this text: "...you can select from those four servers to see the URI for this endpoint at each."
Can you clarify?